### PR TITLE
travis: PHP 7.0 nightly added

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ php:
     - 5.4
     - 5.5
     - 5.6
+    - 7.0
```
Comment (jubianchi) [inline .travis.yml]: PHP7 should be an allowed failure until it gets stable

Comment (Hywan) [inline .travis.yml]: :+1:

Comment (TomasVotruba) [inline .travis.yml]: @jubianchi Added.

Also I fixed standard to 2 spaces (like commands bellow). If you prefer 4 just let me know.

**Comment by jubianchi** [comment]
:+1:

**Comment by pierrejoye** [comment]
Thanks for that, we need it :)

Also we should make sure to report bugs if we see something breaking pickle
in 7, if not part of the small BC breaks already listed.

**Comment by Hywan** [comment]
Thanks @TomasVotruba!

**Comment by TomasVotruba** [comment]
Thank you!

**Comment by pierrejoye** [comment]
Btw do we have hhvm on travis? :)

**Comment by pborreli** [comment]
yes hhvm (3.5.0 right now) and hhvm-nightly

**Comment by TomasVotruba** [comment]
@pierrejoye Shall I send PR with both having nightly in allowed failure?

**Comment by pborreli** [comment]
@TomasVotruba  `hhvm` and `hhvm-nightly` won't work and should be in `allow_failures` section until #100 is open

**Comment by TomasVotruba** [comment]
@pborreli I see. Like this? https://github.com/FriendsOfPHP/pickle/pull/102

**Comment by pborreli** [comment]
@TomasVotruba yes looks good, thanks :+1:

**Pull request merged**